roadside-forms-frontend/frontend_web_app/src/components/Forms/TwentyFourHourForm/component/prohibition.js

```diff
@@ -0,0 +1,99 @@
+import { Radio } from "../../../common/Radio/radio"
+import { Input } from "../../../common/Input/Input"
+import { SearchableSelect } from "../../../common/Select/SearchableSelect"
+import { DatePickerField } from '../../../common/DateField/DatePicker';
+import { TimeInputField } from '../../../common/Input/TimeInputField';
+import { NumericInput } from "../../../common/Input/NumericInput";
+import { useFormikContext } from 'formik';
+import { Checkbox } from "../../../common/Checkbox/checkbox";
+import PropTypes from 'prop-types';
+
+export const Prohibition = (props) => {
+    const { values } = useFormikContext();
+    const {cities} = props;
+    return (
+        <div className='border-design-form left text-font'>
```

Comment:
For the path of this file is there a reason there is another component folder? I just dont want to make the folder structure any more complicated than it needs to be.

Reply:
No, there is no code related reason its just made sense to have one main file outside like for twentyHourForm folder, have one main file and have smaller files inside that component which are getting rendered on main file. if you think we don't need it, thats fine for me